modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -102,6 +113,10 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 	var transport http.RoundTripper
 	var transportModuleName string
 
+	// collect the response matchers defined as subdirectives prefixed with "@"
+	// for use with "handle_response" blocks
+	responseMatchers := map[string]caddyhttp.ResponseMatcher{}
```

Comment:
This is a nit, to be sure, but I get a little triggered when I see a `map` without a `make()` around it. (Am I the only one? And yes I know both are perfectly valid.)

```suggestion
	responseMatchers := make(map[string]caddyhttp.ResponseMatcher)
```

Reply:
:man_shrugging: I have absolutely no preference